src/ui.form-input/index.vue

```diff
@@ -57,10 +66,17 @@
         />
       </label>
 
-      <span v-if="hasSlotContent($slots['right']) || iconRight" v-bind="rightSlotAttrs">
-        <!-- @slot Use it to add icon after text. -->
-        <slot name="right">
-          <UIcon v-if="iconRight" :name="iconRight" />
+      <!-- @slot Use it to add something after text. -->
+      <slot name="right" />
+
+      <span v-if="hasSlotContent($slots['icon-right']) || iconRight" v-bind="rightIconSlotAttrs">
+        <!--
+          @slot Use it to add icon after text.
```

Comment:
@slot Use it to add icon after **the** text. - please fix it in every MR you already open.